src/soundLoop.js

```diff
@@ -6,12 +6,17 @@ define(function (require) {
 
   /**
    * SoundLoop
-   *
+   * Beats are the periodic and repeating fluctuations heard in the intensity of a sound when two sound waves of very similar frequency interfere with one another.
+   * Nodes are the locations where no sound is produced as a result of destructive interference.
+   * Beat pattern is characterized by a wave whose amplitude is changing at a regular rate.
```

Comment:
this is describing a different type of beat pattern that has to do with sound waves. We're talking about a step sequencer pattern, something like this https://drumbit.app/